lib/rubygems/commands/fetch_command.rb

```diff
@@ -42,15 +46,27 @@ def usage # :nodoc:
     "#{program_name} GEMNAME [GEMNAME ...]"
   end
 
+  def check_version # :nodoc:
+    if options[:version] and options[:version] != Gem::Requirement.default and
```

Comment:
I noticed that `check_version` in the install command does not check whether `options[:version]` was passed at all, and instead set `Gem::Requirement.default` as its the default value in `initialize`. I guess we can do the same for consistency.